src/app/static/src/app/store/load/actions.ts

```diff
@@ -144,14 +144,17 @@ const getRehydrateResult = async (context: ActionContext<LoadState, RootState>)
         const {files, savedState} = constructRehydrateProjectState(rootState, response.data)
 
         if (!rootGetters.isGuest) {
-            await (dispatch("projects/createProject",
+            await dispatch("projects/createProject",
                 {
                     name: state.projectName,
                     isUploaded: true
-                }, {root: true}));
+                }, {root: true});
 
-            savedState.projects.currentProject = rootState.projects.currentProject
-            savedState.projects.currentVersion = rootState.projects.currentVersion
+            const project = {
+                currentProject: rootState.projects.currentProject,
+                currentVersion: rootState.projects.currentVersion
+            }
+            Object.assign(savedState.projects, project)
         }
```

Comment:
I tried this and it fixed the problem for me. `emptyState` is imported from `root.ts`
```suggestion
            savedState.projects!.currentProject = rootState.projects.currentProject
            savedState.projects!.currentVersion = rootState.projects.currentVersion
            const newRootState = {
                ...emptyState(),
                ...savedState
            }
            Object.assign(rootState, newRootState);
        }
```

Reply:
Unfortunately I can't test the behaviour. I has added fixes but I am unable to reproduce the state bug, hoping all is fixed as suggested.